### DISCUSSION TITLE
Library directory won't update

### DISCUSSION THREAD

**Discussion by TheGeneCode** [TV-Rename/tvrename#798] [Q&A] [answered]
I'm migrating my media library. I need to point to a new Drive and folder location. I updated the Library Folders section, deleting the old and adding the new, but when I scan it always wants to copy/move to the old drive. I don't understand. Is there some other place I'm supposed to update that I can't find?

**Comment by SirSparkles** [comment, marked as answer]
Each show saves its location as part of its configuration, so you you could do a 'Edit Show xxx' and update the path for the show individually.

Probably better to take your `TVRenameSettings.xml` file in `C:\Users\user\AppData\Roaming\TVRename\TVRename\2.1` and:
1. Take a backup
2. Edit with a text editor
3. Search and replace - you are looking for the `<FolderBase>` tags

Hope that helps

**Reply by TheGeneCode** [reply]
Yay! Thank you Sir Sparkles. /bow
